src/configuration/Configuration.cpp

```diff
@@ -236,8 +268,11 @@ namespace SlimeVR {
                         continue;
                     }
 
+                    m_Logger.trace("Found calibration data file: %s", f.name());
+
                     CalibrationConfig calibrationConfig;
                     f.read((uint8_t*)&calibrationConfig, sizeof(CalibrationConfig));
```

Comment:
```suggestion
                    f.read(reinterpret_cast<uint8_t*>(&calibrationConfig), sizeof(CalibrationConfig));
```

Reply:
Same thing here

Reply:
compiles fine on both platforms, reopening

Reply:
Can we please not touch I didn't play around in? that's what I meant with "Same thing here"